app/components/elements/forms/non_repeatable_nested_component.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Elements
+  module Forms
+    # Encapsulates a non-repeatable nested form.
```

Comment:
Can you add a comment like: "Don't call this directly, use NestedComponentPresenter"?

Reply:
Good call, yes, done.